HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/InvoiceTableViewCellModel.swift

```diff
@@ -39,4 +46,37 @@ final class InvoiceTableViewCellModel {
     var isRecipientLabelHidden: Bool {
         recipientNameText.isEmpty
     }
+    
+    var shouldShowPaymentComponent: Bool {
+        invoice.isPayable ?? false
```

Comment:
I added the Bool as optional because the SDK's consumer needs to call `checkIfDocumentIsPayable` to populate this field. But, I can make it non-optional.